tedge_modbus/operations/update_coil.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+"""Modbus Write Coil Status operation handler"""
+import logging
+
+from pymodbus.exceptions import ConnectionException
+from .context import Context
+from .common import (
+    parse_json_arguments,
+    prepare_client,
+    apply_loglevel,
+    close_client_quietly,
+)
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+)
+
+
+def run(arguments: str | list[str], context: Context) -> None:
+    """Run update_coil operation handler
+    Expected arguments (JSON):
+    {
+        "id": < operationId >,
+        "c8y_SetCoil": {
+           "ipAddress": < ipaddress or empty >,
+          "address": < Fieldbusaddress >,
+          "coil": < coilnumber >,
+          "value": < 0 | 1 >}
+    }
+    Parse JSON payload"""
+    payload = parse_json_arguments(arguments)
+
+    # Load configs and set log level
+    modbus_config = context.base_config
+    apply_loglevel(logger, modbus_config)
+    logger.info("New update_coil operation")
+
+    try:
+        ops = payload["c8y_SetCoil"]
```

Comment:
With `${.payload.c8y_SetCoil}` in `c8y_UpdateCoil` file, you just have to remove this line.